src/main/java/Miki.java

```diff
@@ -0,0 +1,59 @@
+import command.Command;
+import shigure.Ui;
+import storage.Storage;
+import task.Parser;
+import task.TaskList;
+
+import java.io.IOException;
+
+/**
+ * Main class of the Miki personal-task manager project.
+ * In honor of Fuzuki Miki.
+ */
+public class Miki {
+    /**
+     * Main function of the Miki project.
+     *
+     * @param args command-line arguments from the initial program call.
+     */
+    public static void main(String[] args) {
+        boolean hasAsciiOnly = false;
+        boolean hasNoAutoload = false;
+        for (int i = 0; i < args.length; i++) {
+            if (args[i].equals("--ascii-only")) {
+                hasAsciiOnly = true;
+            }
+            if (args[i].equals("--no-autoload")) {
+                hasNoAutoload = true;
+            }
+        }
+
+        Ui ui = new Ui(hasAsciiOnly);
+        Storage storage = new Storage("./data/");
+        TaskList tasks = new TaskList();
+        ui.printIntro();
+
+        if (!hasNoAutoload) {
+            try {
+                storage.load("autosave.txt", tasks);
+            } catch (IOException | Storage.MikiLoadException ex) {
+
```

Comment:
Would it be better if the user can see that the loading has error?

Reply:
True - no message is raised if there's no autosave file since that's 'normal' when we're running for the first time,
but now if the file exists but is corrupt there's a new message to alert the user.